cluster/nats_rpc_client.go

```diff
@@ -155,7 +156,21 @@ func (ns *NatsRPCClient) Call(
 		err = constants.ErrRPCClientNotInitialized
 		return nil, err
 	}
-	req, err := buildRequest(ctx, rpcType, route, session, msg, ns.server)
+
+	if session != nil {
+		requestID := uuid.New().String()
+		requestInfo := ""
+		if route != nil {
+			requestInfo = route.Method
+		}
+
+		session.SetRequestInFlight(requestID, requestInfo, true)
+		defer session.SetRequestInFlight(requestID, "", false)
+	}
+
+	logger.Log.Debugf("sending remote nats request (client) with timeout of %s", ns.connectionTimeout)
```

Comment:
do we need this log line? we already have log for sending requests

Reply:
we don't, same as above, left from debug testing.. removing it

Reply:
Apparently we don't. This was useful for me to figure out which part of the request things were hanging up on, although the line isn't very insightful (the route for example and some request ID would be helpful on this log)

Reply:
![image](https://user-images.githubusercontent.com/12141667/214286757-c82e9fa1-2627-4748-90fc-a78da5d6e26e.png)
Here's an example log with debug enabled.. there's nothing logging the request